Trigger a build to test new start-moosebuild.sh

milljm opened the pull request.

Commit:
Trigger a build to test new start-moosebuild.sh

moosebuild commented:
All [jobs](https://civet.inl.gov/event/36835/) on 470ce32 : invalidated by @milljm

updated civet recipes... restart

Commit:
Force conda to install PyQt=5.9.2

There is a segfault when import PyQt5, for version: 5.6.0.
Qt might also be effected. So lets install a known good combo (Qt=5.9.6)

moosebuild commented:
All [jobs](https://civet.inl.gov/event/36906/) on db4b23f : canceled by @milljm

something wrong with this commit...
